Dockerfile.fedora

```diff
@@ -0,0 +1,18 @@
+FROM fedora:30
+
+RUN yum install -y \
+    gcc-c++ \
+    java-1.8.0-openjdk-devel \
+    java-1.8.0-openjdk-openjfx-devel \
+    make \
+    maven \
+    spdlog-devel \
+    which && \
+  mkdir -p /usr/java/latest/jre && \
+  ln -s /usr/lib/jvm/openjfx/rt/lib /usr/java/latest/jre/lib
```

Comment:
Maybe these should be separate commands instead of being grouped with the install?

Reply:
This is a copy/paste/tweak from the original Dockerfile - as soon as JavaFX gets removed, we no longer need to install `openjfx` or create the symlink - are you happy for this to stay as-is, or shall I change it (and the original Dockerfile to match)?

Reply:
Oh. Seems fine to keep it, then.